debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -234,7 +234,6 @@ public static void j9bcutil_dumpRomMethod(PrintStream out, J9ROMMethodPointer ro
 	 *			-ACC_TRANSIENT
 	 *			-ACC_SYNTHETIC
 	 *			-ACC_ENUM
-	 *			-ACC_STRICT_INIT
```

Comment:
This should stay; suggest adding a constant (perhaps in this class):
```java
	private static final int CFR_ACC_STRICT_INIT = J9CfrClassFile.CFR_ACC_STRICT;
```
The references on lines 437 and 439 should use that new constant.